site/content/blog/markdowndb-launch.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Announcing the official launch of MarkdownDB! 🚀
+description: We're excited to announce the official launch of MarkdownDB, an open library to transform markdown content into SQLite database. And now, it has a dedicated website at markdowndb.com
+date: 2023-10-11
+authors: ['Ola Rubaj']
+filetype: 'blog'
```

Comment:
does the `filetype` do anything? I've not found it working on files outside of blog directory?

Reply:
Can be removed